views/index.erb

```diff
@@ -1,41 +1,99 @@
 <% if @profile %>
-  <h3><%= @profile["first_name"] %>'s Amazing Profile</h3>
-
-  <!-- TODO- Perhaps re-jig this to fit your new design / layout -->
-  <% @profile.each do |field, value| %>
-    <div class="profile-section box">
-      <h4><%= field %></h4>
-      <p class="profile-value"><%= value %></p>
-    </div>
-  <% end %>
-
-  <div id="stats" class="box">
-    <h3>Stats</h3>
-    <!-- TODO - Set the contents of this span with JavaScript -->
-    <span id="word-count">0</span> words found.
+<header id="header-wrap" class="container-fluid">
+  <div id="profile-photo" class="col-md-12 hidden-xs">
+      <a href="http://<%= @profile["homepage"] %>" target="_blank"><img class="profile-img" src="/images/137bb71.jpg" alt="Chris" /></a>
   </div>
+  <div id="details" class="row">
+      <h1 id="name-header" class="text-center"><a href="http://<%= @profile["homepage"] %>" target="_blank"><%= @profile["first_name"] %> <%= @profile["last_name"] %></a></h1>
```

Comment:
You've applied your indentation somewhat inconsistently throughout this template. Did your editor auto-format or was this intentional?